### PR TITLE
Add debug mode to Grakn executable (shell only)

### PR DESCRIPTION
**Pull request by alexjpwalker**
## What is the goal of this PR?

To give users (and ourselves!) a convenient way of running Grakn in debug mode, where Grakn and RocksDB assertions are enabled.

## What are the changes implemented in this PR?

Add debug mode to Grakn executable (shell only, so not Windows).

### CODE REVIEW

**Comment by alexjpwalker** [comment]
This should probably be tested before merging.

**Comment by haikalpribadi** [comment]
@lolski should review this and be set as code owner for this `//bin` package too. @alexjpwalker can you add ganesh into `CODEOWNER` file?

**Comment by alexjpwalker** [comment]
> @lolski should review this and be set as code owner for this `//bin` package too. @alexjpwalker can you add ganesh into `CODEOWNER` file?

Done!

File: .github/CODEOWNERS
Comment (haikalpribadi) [inline .github/CODEOWNERS:1]: No, I meant just give `//bin` to @lolski , @alexjpwalker

**Review by haikalpribadi — CHANGES_REQUESTED** [review]

File: .github/CODEOWNERS
Comment (alexjpwalker) [reply .github/CODEOWNERS:1]: Ahh, should have read the whole message

Comment (alexjpwalker) [reply .github/CODEOWNERS]: fixed

**Review by haikalpribadi — APPROVED** [review]
Approving just for the code owner file. @lolski should review the script.

File: binary/grakn
Comment (lolski) [inline binary/grakn:66]: if we simplify the while loop to this, will it work?
```
if args[3] == '--debug':
  DEBUG=yes
```

or will it clash with our other flags, such as `grakn server start --benchmark`?

File: binary/grakn.bat
Comment (lolski) [inline binary/grakn.bat:38]: When are we adding the debug flag in Windows?

File: binary/grakn
Comment (alexjpwalker) [reply binary/grakn:66]: It would definitely clash with other flags if we did that - we don't want to enforce that --debug is in a particular position, just that it is there.

File: binary/grakn.bat
Comment (alexjpwalker) [reply binary/grakn.bat:38]: I'll raise an issue.

Comment (alexjpwalker) [reply binary/grakn.bat:38]: https://github.com/graknlabs/common/issues/37

File: binary/grakn
Comment (lolski) [reply binary/grakn:66]: Right ok

**Review by lolski — APPROVED** [review]

**Pull request merged**
